doc_source/reference_policies_iam-condition-keys.md

```diff
@@ -250,5 +250,5 @@ You can use the following condition keys in IAM role trust policies for roles th
 
 **sts:ExternalId**  
 Works with [string operators](reference_policies_elements_condition_operators.md#Conditions_String)\.  
-A unique identifier that might be required when you assume a role in another account\. If the administrator of the account to which the role belongs provided you with an external ID, then provide that value in the `ExternalId` parameter\. This value can be any string, such as a passphrase or account number\. Because a cross\-account role is usually set up to trust everyone in an account, the administrator of the trusting account might send an external ID to the administrator of the trusted account\. That way, only someone with the ID can assume the role, rather than everyone in the account\. For more information about the external ID, see [How to Use an External ID When Granting Access to Your AWS Resources to a Third Party](id_roles_create_for-user_externalid.md)\.  
-The `ExternalId` value must have a minimum of 2 characters and a maximum of 1,224 characters\. The value must be alphanumeric without white space\. It can also include the following symbols: plus \(\+\), equal \(=\), comma \(,\), period \(\.\), at \(@\), colon \(:\), forward slash \(/\), and hyphen \(\-\)\.
\ No newline at end of file
+A unique identifier that might be required when you assume a role in another account\. If the administrator of the account to which the role belongs (the trusting account) provided you with an external ID, then provide that value in the `ExternalId` parameter\. This value can be any string, such as a phrase or unique customer identifier\. Because a cross\-account role is usually set up to trust everyone in a trusted account, the administrator of the trusting account might send an external ID to the administrator of the trusted account\. Together with other practices, requiring an external ID helps the trusted account to avoid assuming a role in the trusting account outside of an appropriate context\. For more information about the use and significance of using an external ID, see [How to Use an External ID When Granting Access to Your AWS Resources to a Third Party](id_roles_create_for-user_externalid.md)\.  
```

Comment:
@edenhochbaum @stephswo 

I'd actually take out the bit about:

> Because a cross\-account role is usually set up to trust everyone in a trusted account, the administrator of the trusting account might send an external ID to the administrator of the trusted account\. Together with other practices, requiring an external ID helps the trusted account to avoid assuming a role in the trusting account outside of an appropriate context\.

Which although now doesn't as explicitly suggest insecure behavior, might still be a bit confusing, and the whole "Because a cross\-account role is usually set up to trust everyone in a trusted account" isn't necessarily true, and the causality there is dubious (confused deputies are just as problematic if you trust a single specific principal). It seems more straightforward to say something like:

> This value should be used when a vendor account needs to assume a role in your account, to prevent other customers from misusing that vendor to gain access to your account. This class of behavior is known as a confused deputy attack. For more information ... see ...